### ISSUE TITLE
Unstyled page when running in dev mode

### ISSUE THREAD

**Issue by riyadhzen** [yoonic/nicistore#24]
Hi
I realised that when running in dev mode using "npm run dev", when I navigate to localhost:3000 I get a non styled html page. This is not the case when running in "prod" mode using "npm run prod".

Is this intentional or am i missing something.

**Comment by Hu4F00d** [comment]
I'm having the same issue, can't figure out what's the problem.

**Comment by mikdatdogru** [comment]
same issue, node-sass not installing

**Comment by ekonotek** [comment]
Same issue, did anybody figured it out?
Indeed node-sass isn't even a dependency.
I'll appreciate it.

**Comment by phoenixvk** [comment]
Use the same versions of node and libs that are mentioned there you will
resolve the issue. But the most important issue is nobody here is able to
run API server in production expt.- Author. If anyone got success please
ping me. This is really nice project again.

**Comment by mikdatdogru** [comment]
node: v8.5.0
npm : 5.3.0
i upgraded to  **"node-sass": "4.5.3"** and it's working now. omg im really happy :) 

yes phoenixvk this is really nice project.

**Comment by phoenixvk** [comment]
^^. Ping me if u run can run atlas to production. email. :
vaibhav.kadam45@gmail.com

**Comment by jaivintricore** [comment]
@phoenixvk I am not able to run project in my production server, It has missing css and also issue in connection of atlas
GET http://0.0.0.0:3001/dist/bundle.js net::ERR_CONNECTION_REFUSED
GET http://localhost:8000/v1/account net::ERR_CONNECTION_REFUSED

I have installed app on my production server and to access app through web I have set virtualhost using proxy and reverse proxy and point it to my domain name.

But 0.0.0.0 and localhost connection not working so do you have any workaround to fix this issue?

**Comment by phoenixvk** [comment]
@jaivintricore Yes, I had success with fronted. But I was not able to run backend (atlas) on production. Tried with everything(docker,pm2,nginx(Plain setup)) It didn't worked out for me. And yes I was able to run it in dev. Also their is no issue with CSS it was fine when last time I used it. Just use correct versions on node that author have stated.

I hope this link might help :[https://github.com/yoonic/atlas/issues/19]( https://github.com/yoonic/atlas/issues/19)

**Comment by jaivintricore** [comment]
@phoenixvk thank you for your response.

What do you mean by "And yes I was able to run it in dev" ? development environment of app? or in localhost? 

I am using node v4.8.7 and npm 2.15.11 as suggested, Do I need to use higher version?

Here is how it looks for me. 0.0.0.0 is missing. Same happening while connecting atlas.

https://www.awesomescreenshot.com/image/3112309/93e7209546c237ed58442d4f9a3d65df

**Comment by estebanmuruzabal** [comment]
same problem over here guys... for 8 hours I have been trying stuff but nothing works. did you have any luck?

**Comment by Max7St** [comment]
maybe you just forgot to execute - npm run build
